api/src/pcapi/core/offerers/api.py

```diff
@@ -278,10 +279,13 @@ def validate_offerer(token: str) -> None:
         )
 
 
-def save_venue_banner(user: User, venue: Venue, content: bytes, content_type: str, file_name: str) -> None:
+def save_venue_banner(
+    user: User, venue: Venue, content: bytes, content_type: str, file_name: str, crop_params: tuple = None
```

Comment:
je pense que le typage réel ressemble à quelque chose comme ça

```suggestion
    user: User, venue: Venue, content: bytes, content_type: str, file_name: str, crop_params: Optional[tuple[Optional[pydantic.confloat(ge=0.0, le=1.0)], Optional[pydantic.confloat(ge=0.0, le=1.0)], Optional[pydantic.confloat(ge=0.0, le=1.0)]] = None
```

généralement quand ça devient aussi complexe, dans d'autres langages, j'extrais le type vers quelque chose mis en commun (je ne sais pas si on peut le faire en Python)

Reply:
Avec **Pydantic**, tout est possible 💖